README.md

```diff
@@ -89,9 +104,9 @@ The updates should then be live: <http://drill.apache.org>.
 
 The documentation pages are under `_docs`. Most files, in the YAML front matter, have three important parameters:
 
-* title: - This is the title of the page enclosed in quotation marks. Each page must have a *unique* title
-* date: - This field is needed for Jekyll to write a last-modified date. Initially, leave this field blank.
-* parent - This is the title of the page's parent page. It should be empty for top-level sections/guides, and be identical to the title attribute of another page in all other cases.
+* `title:` - This is the title of the page enclosed in quotation marks. Each page must have a *unique* title
+* `date:` - This field is needed for Jekyll to write a last-modified date. Initially, leave this field blank.
+* `parent:` - This is the title of the page's parent page. It should be empty for top-level sections/guides, and be identical to the title attribute of another page in all other cases.
```

Comment:
Does it make sense to move the build stuff after this intro? I wondered what "title:" and so on were above. Then, when I get here, I now learn what they are. Would be great to reverse the order.

Reply:
Good point! Moved this section above `# Developing and Previewing the Website`